docs/example outputs.md

```diff
@@ -0,0 +1,1083 @@
+This file contains example outputs from nikss-ctl command under different options and externs.
+
+# Error, warning and informational messages
+
+All informational, error and warning messages are printed to the `stderr` stream (with exceptions, see note below). Such
+sentence has a form of sentence with additional context and decoded error code value if available. Some sentences starting
+with keyword `libbpf:` are generated by `libbpf` library itself. When error occurs code other than `0` is returned and
+this code is related to standard `errno` values.
+
+There is no used prefix to point severity of the output sentence.
```

Comment:
??

Reply:
Maybe `log level` would be better term. I mean that messages printed by the library has no information they are `error`, `warning`, `debug`, `info` or other level of importance, so you have to read whole message and understand it to know whether it is important or not.

Some messages start with `warning:` or `error` but most of messages not.